go/libkb/rpc_exim.go

```diff
@@ -958,6 +959,9 @@ func publicKeyV2BaseFromComputedKeyInfo(kid keybase1.KID, info ComputedKeyInfo)
 			FirstAppearedUnverified: info.FirstAppearedUnverified,
 			SigningKID:              info.RevokedBy,
 		}
+		if info.RevokedAtSigChainLocation != nil {
+			base.Revocation.SigChainLocation = *info.RevokedAtSigChainLocation
+		}
```

Comment:
Is it weird that `RevokedAt` could be non-nil but `RevokedAtSigChainLocation` could be nil? Or vice versa? We could consider making them two fields on a struct, behind a single pointer? But this is super nitpicky, so only if you feel like you want to do it.